src/tasks/publish-message-task.ts

```diff
@@ -55,10 +55,15 @@ export class PublishMessageTask extends BaseChatTask<ChatMessage> {
     // publish message
     await this.preHookHandler?.(chatMessage, this.actor, { log, handler });
     const res = await this.chatService.publishMessage(chatMessage, handler);
-    await this.postHookHandler?.(res, this.actor, { log, handler });
+    const newChatMessage = {
+      ...res,
+      // replace updatedAt from null value to empty string
+      ...(res?.updatedAt ? null : { updatedAt: '' }),
```

Comment:
Why do you need to do that?

Reply:
@pyphilia  When a message is created in the database, the `updatedAt` field is `null`. This is fine in the database but i do not like that it returns: 
```json
{
    "id": "4c203a5c-b755-4bb2-8125-8b60b8988b09",
    "chatId": "d6898fca-8c70-4a5b-a3f7-68f503b53572",
    "creator": "a1112eb7-7f28-4fdb-92cc-28171177463f",
    "createdAt": "2022-04-19T09:51:17.484Z",
    "updatedAt": null,
    "body": "Dha message !"
}
```

Instead i would prefer that it returns:
```json
{
    "id": "4c203a5c-b755-4bb2-8125-8b60b8988b09",
    "chatId": "d6898fca-8c70-4a5b-a3f7-68f503b53572",
    "creator": "a1112eb7-7f28-4fdb-92cc-28171177463f",
    "createdAt": "2022-04-19T09:51:17.484Z",
    "updatedAt": "",
    "body": "Dha message !"
}
```

So maybe there is a more elegant way of doing it but i don't know...